create_infotexts.py

```diff
@@ -168,24 +172,25 @@ def create_smvk_mm_link(item):
 
 
 def select_best_mapping_for_depicted_person(flipped_name):
+    """Lookup available mappings for a string respresenting a name and return best choice.
+
+    :flipped_name: string representing full name turned around from e.g. "surname, given name" -> "given name surename"
```

Comment:
Linebreak (at <80 characters) and indent the remainder. I.e.
```
:flipped_name: string representing full name turned around from
    e.g. "surname, given_name" -> "given_name surname"
```
Also one typo and clarifying that given name is often one word.